source/changelog.txt

```diff
@@ -24,6 +24,18 @@ v2.0.0
   - ``db.enableFreeMonitoring``
   - ``db.disableFreeMonitoring``
 
+- The following :data:`config.version` fields were removed and
```

Comment:
(wording nit)
Maybe something like:
The following config.version fields were removed and won't be returned in sh.status() output.

Reply:
Will do.